docker/wait-for-it.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# wait until MySQL is really available
+maxcounter=45
```

Comment:
I'd make maxcounter an env with a descriptive name and set it in docker-compose.